restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/provisioning/number/vi/IncomingPhoneNumbersEndpointTest.java

```diff
@@ -1004,6 +1007,65 @@ public void testAccountAssociatedPhoneNumbersSecondFilter() {
         assertTrue(clientResponse.getStatus() == 204);
     }
     
+    @Test
+    public void testPurchasePhoneNumberValidation() {
```

Comment:
Hi @muhammadbilal19 thanks for adding test case.

Lets say we have following 4 variations of a number:

1. withoutPlusOrZeros (923212345678)
2. withPlus (+923212345678)
3. withDoubleZero (00923212345678)
4. withPlusAndDoubleZero (+00)

Here are possible use cases:

1. Existing number in DB is in format 1, try adding same number with format 1,2,3,4.
2. Existing number in DB is in format 2, try adding same number with format 1,2,3,4.
3. Existing number in DB is in format 3, try adding same number with format 1,2,3,4.
4. Existing number in DB is in format 4, try adding same number with format 1,2,3,4.

Following test case tests use case number 2.
Can you please cover other 3. it would be pretty similar to what you already have.

Please share if you think otherwise.

Regard,
Maria